lib/utils/register_provider.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:clnapp/api/api.dart';
+import 'package:clnapp/api/client_provider.dart';
+import 'package:clnapp/api/cln/cln_client.dart';
+import 'package:clnapp/model/user_setting.dart';
+import 'package:clnapp/utils/app_provider.dart';
+
+class RegisterProvider {
+  void registerClientFromSetting(Setting setting, AppProvider provider) {
```

Comment:
```suggestion
  static void registerClientFromSetting(Setting setting, AppProvider provider) {
```

So you can call it like `RegisterProvider.registerClientFromSetting(...);`